src/uwtools/drivers/driver.py

```diff
@@ -220,6 +220,17 @@ def run(self):
         yield self._taskname("run")
         yield (self._run_via_batch_submission() if self._batch else self._run_via_local_execution())
 
+    @task
+    def runscript(self):
+        """
+        The runscript.
+        """
+        path = self._runscript_path
+        yield self._taskname(path.name)
+        yield asset(path, path.is_file)
+        yield None
+        self._write_runscript(path)
```

Comment:
This was repeated verbatim in many driver modules, so I factored it out here as a base-class default. Some drivers override it.